server_admin/topics/clients/oidc/con-basic-settings.adoc

```diff
@@ -77,3 +75,19 @@ Domain URLs listed here are embedded within the access token sent to the client
 
 [[_back-channel-logout-url]]
 *Backchannel Logout URL*:: URL that will cause the client to log itself out when a logout request is sent to this realm (via end_session_endpoint). If omitted, no logout requests are sent to the client.
+
```

Comment:
At line 70, insert this content

ifeval::[{project_community}==true]

== Authentication flow

* *Standard Flow*
+
When enabled, clients can use the OIDC xref:_oidc-auth-flows-authorization[Authorization Code Flow].

* *Direct Access Grants*
+
When enabled, clients can use the OIDC xref:_oidc-auth-flows-direct[Direct Access Grants].

* *Implicit Flow*
+
When enabled, clients can use the OIDC xref:_oidc-auth-flows-implicit[Implicit Flow].

* *Service account roles*
+
When enabled, allows this client to authenticate to {project_name} and retrieve access token dedicated to this client. In terms of OAuth2 specification, this enables support of 'Client Credentials Grant' for this client.

* *Auth 2.0 Device Authorization Grant*
+
If enabled, clients are allowed to use the OIDC xref:con-oidc-auth-flows_server_administration_guide[Device Authorization Grant].

* *OIDC CIBA Grant*
+
If enabled, clients are allowed to use the OIDC xref:con-oidc-auth-flows_{context}[Client Initiated Backchannel Authentication Grant].

== Capability Config

* *Root URL*
+
If {project_name} uses any configured relative URLs, this value is prepended to them.

* *Home URL*
+
Provides the default URL for when the auth server needs to redirect or link back to the client.

* *Valid Redirect URIs*
+
Required field.  Enter a URL pattern and click *+* to add and *-* to remove existing URLs and click *Save*. You can use wildcards at the end of the URL pattern. For example $$http://host.com/*$$
+
Exclusive redirect URL patterns are typically more secure.  See xref:unspecific-redirect-uris_{context}[Unspecific Redirect URIs] for more information.

* *Web Origins*
+
Enter a URL pattern and click *+* to add and *-* to remove existing URLs. Click *Save*.
+
This option handles link:https://fetch.spec.whatwg.org/[Cross-Origin Resource Sharing (CORS)].
If browser JavaScript attempts an AJAX HTTP request to a server whose domain is different from the one that the
JavaScript code came from, the request must use CORS. The server must handle CORS requests, otherwise the browser will not display or allow the request to be processed. This protocol protects against XSS, CSRF, and other JavaScript-based attacks.
+
Domain URLs listed here are embedded within the access token sent to the client application. The client application uses this information to decide whether to allow a CORS request to be invoked on it.  Only {project_name} client adapters support this feature. See link:{adapterguide_link}[{adapterguide_name}] for more information.

[[_admin-url]]
* *Admin URL*
+
Callback endpoint for a client.  The server uses this URL to make callbacks like pushing revocation policies, performing backchannel logout, and other administrative operations.  For {project_name} servlet adapters, this URL can be the root URL of the servlet application.
For more information, see link:{adapterguide_link}[{adapterguide_name}].

endif::[